examples/pipe/dev/user.clj

```diff
@@ -0,0 +1,22 @@
+(ns user
+  (:require [system :refer :all]))
+
+
+(comment
+  ;;; Evaluate the forms.
```

Comment:
I thought it makes more sense to have the sample calls in this ns (rather than `pipe`) since, the readme was referring to the `user` ns.